src/brain/issueLabelHandler/index.ts

```diff
@@ -25,4 +26,6 @@ export async function issueLabelHandler() {
   githubEvents.on('issue_comment.created', updateCommunityFollowups);
   githubEvents.removeListener('issues.labeled', ensureOneWaitingForLabel);
   githubEvents.on('issues.labeled', ensureOneWaitingForLabel);
+  githubEvents.removeListener('projects_v2_item.edited', syncLabelsWithProjectField);
+  githubEvents.on('projects_v2_item.edited', syncLabelsWithProjectField);
```

Comment:
This looks good to me. What do you think of putting the reverse sync in its own handler as well, instead of tightly coupling it to routing? That way we're set up better for `Waiting for` labels.

Reply:
yeah good idea, just separated that into its own handler

Reply:
Ah, hrm. I was actually thinking of factoring out `modifyProjectIssueProductArea` and making it generic, so that it could also be used for "Waiting for" in the future. Could cross that bridge later if you want.

Reply:
Ah got it, I do like having the `projects_v2_item` events in another listener so I think that's a nice change to have when we end up adding more automations in the future. This generic refactor seems trivial enough to implement so I'll do it now.

Reply:
done, made the function generic and named it `modifyProjectIssueField`

Reply:
08d6d49b5fc5f03a8af818439514271875eea516. Looks good enough for now! :)